main/logkeeper.go

```diff
@@ -79,10 +78,10 @@ func main() {
 
 	catcher := grip.NewCatcher()
 	router := lk.NewRouter()
+	router.Use(logkeeper.NewLogger(ctx).Middleware)
 	n := negroni.New()
-	n.Use(logkeeper.NewLogger(ctx))              // includes recovery and logging
 	n.Use(negroni.NewStatic(http.Dir("public"))) // part of negroni Classic settings
-	n.UseHandler(gorillaCtx.ClearHandler(router))
+	n.UseHandler(router)
```

Comment:
To confirm, we're removing the explicit ClearHandler because the Gorilla Mux that we're using as a router should be doing that already, right?

Reply:
Although it's not directly related, I removed the ClearHandler call because it was just a relic. It was added in [this commit](https://github.com/evergreen-ci/logkeeper/commit/a9a77e4160ff3e4e2284f9c655f08f70ddcaac38) to add the request id to the request context. In [this later commit](https://github.com/evergreen-ci/logkeeper/commit/d32925bf1576b601543d958129fe5b1c642607ac) we switched out the gorilla.Context in favor of the (then new) request.Context that was built into go, but we didn't remove this line that was clearing the gorilla context. So we're clearing just clearing [these empty maps](https://github.com/gorilla/context/blob/8559d4a6b87e4f517ec1846eb90a192b8748cc89/context.go#L15-L16) before every request...